### PR TITLE
created wishlist page

### PR DESCRIPTION
**Pull request by sagargse**
Listing of selected products
image

### CODE REVIEW

File: wishlist-page/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ravikumar0403) [inline wishlist-page/wishlist.html:7]: You can add author meta tag

**Review by ravikumar0403 — APPROVED** [review]
commented one suggestion rest all looks good to me

File: wishlist-page/wishlist.html (hunk 1 above)
Comment (sagargse) [reply wishlist-page/wishlist.html:7]: ok will do that

File: wishlist-page/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+<body>
+    <div class="wishlist-grid-container">
+        <nav class="navbar">
+            <ul class="header-container">
+                <li class="logo">KidsZilla</li>
+                <li class="search-bar">
+                    <input type="search" placeholder="search">
+                    <span class="search-icon fa fa-search"></span>
+                </li>
+                <input type="checkbox" id="click">
```
Comment (vishalg8454) [inline wishlist-page/wishlist.html:26]: A `value` attribute should also be present for input of type checkbox.

File: wishlist-page/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                            <p>Multicoloured soft toys Disney</p>
+                            <span class="text-secondary">₹299</span>
+                            <div class="rating-container">
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon checked fa fa-star"></i>
+                                <i class="rating-icon fa fa-star"></i>
+                            </div>
+                            <br>
```
Comment (vishalg8454) [inline wishlist-page/wishlist.html:83]: use of `br` for vertical spacing is not good.

File: wishlist-page/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                            <div class="card-buttons">
+                                <a class="cart-button">Add to Cart</a>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="vertical-card simple-card">
+                        <img src="/assets/toy2.jpg" alt="image">
+                        <div class="card-desc-block">
+                            <h1>Mickey Mouse</h1>
```
Comment (vishalg8454) [inline wishlist-page/wishlist.html:92]: `h1` should be used exclusively in a webpage, that is only once. Replace this with something like `h2` or `h3`

**Review by vishalg8454 — COMMENTED** [review]
Rest all looks fine, just few minor changes needs to be done.

File: wishlist-page/wishlist.html (hunk 3 above)
Comment (sagargse) [reply wishlist-page/wishlist.html:83]: okay! will consider that

File: wishlist-page/wishlist.html (hunk 4 above)
Comment (sagargse) [reply wishlist-page/wishlist.html:92]: okay!

**Pull request merged**